isis/tests/FunctionalTestsMap2Map.cpp

```diff
@@ -0,0 +1,237 @@
+#include <QTemporaryDir>
+
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Histogram.h"
+
+#include "map2map.h"
+
+#include "gtest/gtest.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/map2map.xml").expanded();
+
+
+TEST_F(ThreeImageNetwork, FunctionalTestMap2mapDefault) {
+
+  QString outCubeFileName = tempDir.path() + "/outTemp.cub";
+  QVector<QString> args = {"from=" + cube1map->fileName(),
+                           "to=" + outCubeFileName,
+                           "map=" + cube2map->fileName(),
+                           "defaultrange=map",
+                           "INTERP=NEARESTNEIGHBOR"};
```

Comment:
I think it could be worth doing a histogram check for these test cases. Other than that everything looks good

Reply:
Added histograms. They have strange outputs so I am not sure what's going on.

Reply:
Histograms looks correct when running against the original uncropped test images so this output may be due to using cropped images and trying to use the ThreeImageNetwork fixture. Not sure if this is okay for testing or if we need to stick with using the original test data for these gtests.